### PR TITLE
Added missing translations in Indonesian

### PR DESCRIPTION
**Pull request by Joshua-Ramon-Enslin**
Updated JabRef_in.properties

### CODE REVIEW

**Comment by mlep** [comment]
@Joshua-Ramon-Enslin  : You are so fast! Thank you!

File: src/main/resources/l10n/JabRef_in.properties [hunk 1]
```diff
@@ -1009,289 +1009,289 @@ Style_selection=Pilihan_gaya
...
 Convert_to_BibLatex_format_(for_example,_move_the_value_of_the_'journal'_field_to_'journaltitle')=
-Could_not_apply_changes.=
+Could_not_apply_changes.=Tidak_bisa_terapkan_perubahan.
 Deprecated_fields=
-Hide/show_toolbar=
-No_read_status_information=
-Printed=
-Read_status=
-Read_status_read=
+Hide/show_toolbar=Sembunyikan/Tampilkan toolbar
```
Comment (mlep) [inline src/main/resources/l10n/JabRef_in.properties]: Unwanted space before "toolbar"

File: src/main/resources/l10n/JabRef_in.properties [hunk 2]
```diff
@@ -1009,289 +1009,289 @@ Style_selection=Pilihan_gaya
...
+Importing_cancelled=Impor_dibatalkan
+There_are_no_available_databases_to_be_imported=Tidak_ada_basisdata_yang_nyata_untuk_diimpor
+Import_from_SQL_database=Impor_dari_basisdata_SQL
+Imported_%0_databases_successfully=Impor_%0_basisdata_berhasil
+<_CREATE_NEW_DATABASE_>=<_BUAT_BASISDATA_YANG_BARU_>
+Remove_Selected=Hapuskan_pilihan
+SQL_Database_Exporter=Fungsi_Ekspor_Basisdata_SQL
+Select_target_SQL_database\:=Pilih_basisdata_SQL_target
+SQL_Database_Importer=Fungsi_Impor_Basisdata_SQL
+Please_select_which_JabRef_databases_do_you_want_to_import\:=Silahkan_memilih_basisdata_JabRef_untuk_diimpor/:
```
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_in.properties]: Wrong slash?

File: src/main/resources/l10n/JabRef_in.properties [hunk 3]
```diff
@@ -1009,289 +1009,289 @@ Style_selection=Pilihan_gaya
...
+Use_the_selected_directory_to_start_with_the_search.=Gunakan_direktori_polihan_untuk_memulai_pencarian.
+Opens_the_file_browser.=Buka_penjelajah_berkas
+Scan_directory=Pindai_direktori
+Searches_the_selected_directory_for_unlinked_files.=Mencari_berkas_yang_belum_ditautkan_dalam_direktori_pilihan.
+Starts_the_import_of_BibTeX_entries.=Memulai_impor_entri_BibTeX.
+Leave_this_dialog.=Tinggalkan_dialog_ini.
+Create_directory_based_keywords=Buat_katakunci_berdasar_direktori
+Creates_keywords_in_created_entrys_with_directory_pathnames=Buat_katakunci_dalam_entri_pembuatan_dengan_direktori
+Select_a_directory_where_the_search_shall_start.=Pilih_direktori_mulaan_pencarian.
+Select_file_type\:=Pilih_tipe_berkas:\
```
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_in.properties]: Dash at wrong position?

File: src/main/resources/l10n/JabRef_in.properties [hunk 4]
```diff
@@ -1009,289 +1009,289 @@ Style_selection=Pilihan_gaya
...
+These_files_are_not_linked_in_the_active_database.=Para_berkas_ini_tidak_bertautan_dalam_basisdata_yang_aktif
+Entry_type_to_be_created\:=Tipe_entri_untuk_dibuat
```
Comment (stefan-kolb) [inline src/main/resources/l10n/JabRef_in.properties]: Missing slash?

**Comment by Joshua-Ramon-Enslin** [comment]
Okay, seems like the pull request went through but to the old pull request instead of the actual JabRef. Hope this is okay.
In the new one I fixed the mistakes pointed out above.

**Comment by stefan-kolb** [comment]
There is still one comment left above.

**Comment by stefan-kolb** [comment]
Thanks a lot :smile: :+1:  

LGTM :+1:

**Comment by stefan-kolb** [comment]
Looks like not all needed Keys are there as the build is failing with `allFilesMustHaveSameKeys`.

**Comment by mlep** [comment]
Quality check are very picky... Well, that's why they have been implemented...

I believe the problem comes from here:
Currently, the file has the key:
Removed_all_subgroups_of_group_"0".
It should be:
Removed_all_subgroups_of_group_"%0".

Additionnaly: colon missing in the translation of
Select_target_SQL_database\:=Pilih_basisdata_SQL_target

**Comment by Joshua-Ramon-Enslin** [comment]
The updates here say it's okay now. Hope that suffices. Thanks for your help!

**Pull request merged**